benchmarking/commons/benchmark_main.py

```diff
@@ -36,6 +37,17 @@
 from syne_tune.tuner import Tuner
 
 
+BenchmarkDefinitions = Union[
```

Comment:
Could you add a comment on what the string are?

Also having a union can make things more complicated (in particular if it depends on several cases). Is there a particular reason why we would need it? For instance, is it because you want to be able to run all "yahpo-rbv2" benchmarks?

Reply:
Yes, this allows you to parallelize over benchmarks. The outer dict layer is dealt with in remote launcher, the inner dict layer in the sequential loop over benchmarks in benchmark_main.

This is applied in the next PR. YAHPO rbv2 has to many instances that there is no other way.